src/kv/Rpc.cc

```
@@ -22,6 +52,88 @@ std::shared_ptr<KvConnClient> ConnArray::get()
+            log->warning(getCurrentExceptionMsg("RpcClient scan conns failed: "));
+        }
+    }
+}
+
+void RpcClient::stop()
+{
+    stopped.store(true);
+    scan_cv.notify_all();
+}
+
+void RpcClient::scanConns()
```

Clang (14.0.6)

[error] method 'scanConns' can be made static

(readability-convert-member-functions-to-static,-warnings-as-errors)